src/neighbors.cc

```diff
@@ -792,7 +792,9 @@ peerDigestLookup(CachePeer * p, PeerSelector * ps)
     debugs(15, 5, "peerDigestLookup: peer " << p->host << " says HIT!");
 
     return LOOKUP_HIT;
-
+#else
+    (void) p;
```

Comment:
An alternative could be:
```
template <class T>
inline void unused(T &something)
{
    (void)something;
}
```

would this be better?

Reply:
IMO, no, it would not be better. The `(void)x` mnemonic is common, short, seemingly safe, and should be understood/used by more developers than anything custom we can come up with.

We have UseThisStatic() but it serves a different purpose.

Reply:
ok